CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -1,4 +1,10 @@
-# v1.11.0 - Changelog since v1.10.5
+# v1.11.2 - Changelog since v1.11.1
```

Comment:
This gets generated when a new release is cut. So this file should be reverted.

Reply:
Done